Gifski/VideoValidator.swift

```diff
@@ -188,6 +188,15 @@ struct VideoValidator {
 		do {
 			let trimmedAsset = try newAsset.trimmingBlankFramesFromFirstVideoTrack()
 			return .success(trimmedAsset, newVideoMetadata)
+		} catch AVAssetTrack.VideoTrimmingError.codecNotSupported {
+			NSAlert.showModal(
+				for: window,
+				title: "Could not trim empty leading frames from video.",
+				message: "We can't trim empty leading frames from this codec, sorry. You can still create a GIF."
+			)
```

Comment:
Just thinking out loud, do we even need to show this modal? The user cannot do anything about it anyway and it's likely they won't notice a difference, and they will also most likely not understand what it means. Maybe we should just skip the modal?

Reply:
Sounds good to me. 👍 

Removed in 0f15040.